posthog/utils.py

```diff
@@ -187,11 +187,11 @@ def render_template(template_name: str, request: HttpRequest, context=None) -> H
         context["git_rev"] = get_git_commit()
         context["git_branch"] = get_git_branch()
 
-    if settings.DEBUG:
-        context["js_posthog_host"] = "window.location.origin"
+    if settings.SELF_CAPTURE:
+        context["posthog_capture_host"] = "window.location.origin"
     else:
-        context["js_posthog_apikey"] = "sTMFPsFhdP1Ssg"
-        context["js_posthog_host"] = "'https://app.posthog.com'"
+        context["posthog_capture_key"] = "'sTMFPsFhdP1Ssg'"
```

Comment:
You didn't rename this above, and as-is it would cause an error in prod instances.

I think the new name is not strictly better - we're not using this key in BE/it's js-specific.

Reply:
True, will revert to old name